tests/Fixtures/e2e/SymfonyFlex/composer.json

```diff
@@ -0,0 +1,16 @@
+{
+    "require-dev": {
+        "symfony/flex": "^1.1",
```

Comment:
seems like this package does not add any PHPUnit (only in require-dev for testing itself:

https://github.com/symfony/flex/blob/48438c2ef0e9b8d19e861ede35f0118837e6df5c/composer.json#L16-L19

And if this is the case, this test is the same as `tests/Fixtures/e2e/SymfonyBridge`. Shouldn't we test `symfony-website-skeleton` instead?

I've just created project with `website-skeleton`:

```bash
composer show | grep phpunit
symfony/phpunit-bridge              v4.0.4             Symfony PHPUnit Bridge
```

```bash
ls bin
console	phpunit
```